dropshot/src/server.rs

```diff
@@ -271,7 +274,7 @@ impl<C: ServerContext> InnerHttpServerStarter<C> {
                     Ok((sock, remote_addr)) = self.0.accept() => {
```

Comment:
It's still the case in the new code that _if_ the acceptor returns an error, we'll stop trying to call accept.  I guess the idea is that it's only returning persistent errors now?  I think I'd do the thing I suggested elsewhere and have the function _not_ return `Result`.  If we really want to do that, I think we could use a comment here because the issue (and the fact that this is correct) here is really subtle and would be easy to break by accident.